cmd/cluster-node-tuning-operator/main.go

```diff
@@ -247,15 +235,15 @@ func removePerformanceOLMOperator(cfg *rest.Config) error {
 		subscription := &subscriptions.Items[i]
 		subscriptionExists := true
 		for _, csv := range performanceOperatorCSVs {
-			if subscription.Namespace == csv.Namespace && subscription.Status.InstalledCSV == csv.Name {
+			if subscription.Namespace == csv.GetNamespace() && subscription.Status.InstalledCSV == csv.GetName() {
```

Comment:
moving to getters is a bit of sidegrade, but won't hurt

Reply:
Changed.